slides.md

```diff
@@ -854,8 +855,8 @@ export default function (fastify) {
 
 - In the workshop - the database contains the list of leaked passwords in `databreachrecords`
 - Run the server for step 7 (`cd src/a07-authentication-failures`, `npm start`)
-- In Postman, run the query for `A07: Register`. Observe a token is succesfully returned
-- In the database check the `databreachrecords` table for password used in the Postman request body
+- From the browser at `http://localhost:3006`, run the query for `A07: Register`. Observe a token is succesfully returned
```

Comment:
probably we can say at the beginning that our frontend will run on `http://localhost:3006`, and not repeat every time